spec/defines/virtualbox__extpack_spec.rb

```diff
@@ -13,40 +13,26 @@
   sane_defaults = {
     :ensure           => 'present',
     :source           => 'http://server.example.com/extpack.vbox-extpack',
-    :checksum_string  => 'd41d8cd98f00b204e9800998ecf8427e',
+    :checksum		=> 'd41d8cd98f00b204e9800998ecf8427e',
```

Comment:
Arrows don't align - not that I think that's required but the other two lines were aligned specifically.